test/unit/math/gpu/opencl_context_test.cpp

```diff
@@ -0,0 +1,78 @@
+#ifdef STAN_OPENCL
+#include <stan/math/gpu/opencl_context.hpp>
+#include <gtest/gtest.h>
+#include <vector>
+#include <fstream>
+
+TEST(MathGpu, getInfo) {
+  cl::Context cl = stan::math::opencl_context.context();
+  EXPECT_NE("", stan::math::opencl_context.description());
+  EXPECT_NE("", stan::math::opencl_context.capabilities());
+  // auto foo = stan::math::opencl_context.capabilities();
+  // std::cout << foo << std::endl;
+  EXPECT_EQ(256, stan::math::opencl_context.max_workgroup_size());
```

Comment:
Is this going to hold on different computers? We might have to do something else for this test.

Reply:
No, this would not pass on different devices/platforms. The only thing that we can be sure of is that max_workgroup_size should not be zero. I have changed it to EXPECT_NE(0, ...).